Makefile

```diff
@@ -6,10 +6,16 @@ start-anvil-chain-with-contracts-deployed: ##
 	./crates/contracts/anvil/start-anvil-chain-with-el-and-avs-deployed.sh
 
 deploy-eigenlayer:
-	./crates/contracts/anvil/deploy-eigenlayer.sh
+	./crates/operator_sets_contracts/anvil/deploy-eigenlayer.sh
 
 deploy-avs:
-	./crates/contracts/anvil/deploy-avs.sh
+	./crates/operator_sets_contracts/anvil/deploy-avs.sh
+
+deploy-m2-eigenlayer:
+	./crates/m2_contracts/anvil/deploy-eigenlayer.sh
+
+deploy-m2-avs:
+	./crates/m2_contracts/anvil/deploy-avs.sh
```

Comment:
You used these two targets to regenerate the anvil state, right? Could you add a step-by-step guide on how to regenerate it in the readme?

Reply:
done